SphereSurfaceChargeDensity/MathOpsModule.py

```diff
@@ -0,0 +1,23 @@
+""" This Module is to define the general mathematical operations to be used by the other modules """
+
+def div(numerator, denominator):
+    """
+    Returns the result of dividing two numbers
+    Parameters:
+    numerator (float), denominator (float): input numbers
+    Returns:
+    number (float): The result of dividing the first number by the second number utilizing the lambda expression
+    """
+    Div = lambda x, y: x/y
```

Comment:
Do not use your function name and a variable name with different cases. See linting

Reply:
I don't quite get it.

Reply:
Removed.